python/lsst/ts/ATWhiteLightSource/chillerComponent.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 import socket
 import logging
+import time
 
 
 class ChillerComponent(object):
```

Comment:
There's a hard coded IP in here...

Reply:
reading IPs out of the config file is in DM-22043